packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -0,0 +1,138 @@
+/* global before, browser, Terra */
+
+describe('Abstract Modal', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Append Class', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-append-class');
+      browser.setViewportSize(Terra.viewports('medium')[0]);
```

Comment:
Since setViewport is defined in the Abstract Modal before hook, all following instances can of setting the viewport can be removed.

Reply:
I'll remove setViewport from `Abstract Modal` since I want to customize the viewport for `Content Overflow` tests. 

Otherwise the viewport I'm setting in `Content Overflow` will be propagated to all the other tests.

Reply:
Would it make sense for `Content Overflow` steps should set and then readjust the viewport back?